### ISSUE TITLE
prepi missing from parameterize

### ISSUE THREAD

**Issue by ferpozo6** [Acellera/htmd#362]
Hi,

We are trying to use _docking to generate starting poses for simulations_ and when it is going to _build system from docked poses_ our script doesn't find the file called structure.crd. Our code is:
```python
molbuilt = []
for i, p in enumerate(poses):
    prot = Molecule('FXR_APO_structure_D3R_GC2.pdb')
    #prot.filter('chain A and (protein or water or resname CA)')
    prot.set('segid', 'P', sel='protein and noh')
    prot.set('segid', 'W', sel='water')
    prot.center()
    from htmd.molecule.util import maxDistance
    D = maxDistance(prot, 'all')
    
    ligand = p
    ligand.set('segid','L')
    ligand.set('resname','MOL')
    
    mol = Molecule(name='combo')
    mol.append(prot)
    mol.append(ligand)
    
    D = D + 15
    smol = solvate(mol, minmax=[[-D, -D, -D], [D, D, D]])
    caps = dict()
    caps['P'] = ['ACE', 'NME']
    from htmd.builder.amber import defaultParam
    param = defaultParam()
    param.append("param-fxr_1_test/parameters/GAFF2/b3lyp-cc-pVDZ-water/mol.frcmod")
    molbuilt.append(amber.build(smol, param=param, caps=caps, outdir='./docked/build/{}/'.format(i+1), saltconc=0.15))
    if i==1: # For time purposes lets only build the two first
        break
```
```pytb
2017-06-08 18:59:23,370 - htmd.builder.solvate - INFO - Using water pdb file at: /home/carles/miniconda3/lib/python3.6/site-packages/htmd/builder/wat.pdb
2017-06-08 18:59:24,125 - htmd.builder.solvate - INFO - Replicating 8 water segments, 2 by 2 by 2
Solvating: 100% (8/8) [############################################] eta 00:00 \
2017-06-08 18:59:36,002 - htmd.builder.solvate - INFO - After removing water molecules colliding with other molecules, 24318 water molecules were added to the system.
2017-06-08 18:59:37,821 - htmd.builder.amber - INFO - Converting CHARMM membranes to AMBER.
2017-06-08 18:59:43,939 - htmd.builder.amber - INFO - Path frcmod.ionsjc_tip3p not found, assuming a standard AmberTools file.
2017-06-08 18:59:43,940 - htmd.builder.amber - INFO - Writing PDB file for input to tleap.
2017-06-08 18:59:47,078 - htmd.builder.amber - INFO - Starting the build.
2017-06-08 19:00:09,058 - htmd.builder.amber - INFO - Finished building.
---------------------------------------------------------------------------
FileNotFoundError                         Traceback (most recent call last)
<ipython-input-9-6584dc378c7f> in <module>()
     24     param = defaultParam()
     25     param.append("param-fxr_1_test/parameters/GAFF2/b3lyp-cc-pVDZ-water/mol.frcmod")
---> 26     molbuilt.append(amber.build(smol, param=param, caps=caps, outdir='./docked/build/{}/'.format(i+1), saltconc=0.15))
     27     if i==1: # For time purposes lets only build the two first
     28         break

/home/carles/miniconda3/lib/python3.6/site-packages/htmd/builder/amber.py in build(mol, ff, topo, param, prefix, outdir, caps, ionize, saltconc, saltanion, saltcation, disulfide, tleap, execute, atomtypes, offlibraries)
    322         logger.info('Finished building.')
    323 
--> 324         if path.getsize(path.join(outdir, 'structure.crd')) != 0 and path.getsize(path.join(outdir, 'structure.prmtop')) != 0:
    325             molbuilt = Molecule(path.join(outdir, 'structure.prmtop'))
    326             molbuilt.read(path.join(outdir, 'structure.crd'))

/home/carles/miniconda3/lib/python3.6/genericpath.py in getsize(filename)
     48 def getsize(filename):
     49     """Return the size of a file, reported by os.stat()."""
---> 50     return os.stat(filename).st_size
     51 
     52 

**FileNotFoundError: [Errno 2] No such file or directory: './docked/build/1/structure.crd'**
```

We add to the original code of the tutorial only the amber force field build. That lines runs in a proper way when we use it in a _System Building_. 
Do you know what is making crash our program?

We also attach the 'docked/build/1/log.txt' file.
[log.txt](https://github.com/Acellera/htmd/files/1063629/log.txt)

Thanks in advance,
Cheers.
Fernando, Carles and Iván

**Comment by stefdoerr** [comment]
This cannot be the correct `log.txt` file. It's the logfile of `psfgen` meaning `charmm.build` while you are using `amber.build`. Delete the build folder, try again and send me the `tleap.log` and `log.txt` files

**Comment by ferpozo6** [comment]
[logs.zip](https://github.com/Acellera/htmd/files/1063833/logs.zip)

Sorry, these are the right logs.

**Comment by stefdoerr** [comment]
You are missing the `.prepi` file for your ligand called `MOL` (you are only passing the frcmod to the builder). You need both `prepi` (topology) and `frcmod` (parameters) for your to build a system with a ligand.

**Comment by j3mdamas** [comment]
@ferpozo6, I'll let you know when we have a version of parameterize that outputs the prepi

**Comment by mj-harvey** [comment]
You can use a mol2 in place of a prepi

**Comment by j3mdamas** [comment]
Matt is right. The prepi is not necessary and Stefan already fixed the underlying problems in f1baa1e774166606c07a0ad5ab21d739149174d7

@ferpozo6, the issue here is the `poses` list. I assume those are `Molecule` instances of ligands. But, for Amber, they need to be loaded from MOL2 files. Otherwise, `amber.build` will not know they are ligands that needed to loaded as such (to have the correct bonding information). With the next release, which will include the mentioned fix, this is solved.
